### PR TITLE
E0d/forums

### PR DESCRIPTION
**Pull request by e0d**

### CODE REVIEW

**Comment by e0d** [comment]
Tested with this playbook on Ubuntu and Debian

```
- name: Configure instance(s)
  hosts: edx_gce
  sudo: True
  gather_facts: True
  vars:
    migrate_db: True
    mysql5_workaround: True
  roles:
    - common
    - mongo
    - oraclejdk
    - elasticsearch
    - forum
```

File: playbooks/roles/rbenv/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+
+#forum_user: forum
+
+# Ruby Specific Vars
+#ruby_base: /opt/forum
+#rbenv_root: "{{ ruby_base }}/.rbenv"
```
Comment (jarv) [inline playbooks/roles/rbenv/vars/main.yml:7]: unfortunately these names will conflict with ruby vars in edxapp/vars/main.yml. 
Will need to clean up the edxapp role var names.

**Comment by jarv** [comment]
As we discussed the rvm role can take paramters, something like

```
roles:
  - { role: 'rbenv', rbenv_user: 'forum' , rbenv_ruby_version: '1.9.3...' }

```

And then a sanity check in the role:

```
- fail: rbenv_user required for role
  when: rbenv_user is not defined
```

**Comment by e0d** [comment]
Closing in favor of a version against master that has diverged substantially from this PR.

**Pull request closed without merging**